test/helpers/ssh_command.go

```diff
@@ -128,18 +145,27 @@ func (client *SSHClient) RunCommand(cmd *SSHCommand) ([]byte, error) {
 	return session.Output(cmd.Path)
 }
 
-//RunCommandWithTimeout run a command and kill after a timeout
-func (client *SSHClient) RunCommandWithTimeout(cmd *SSHCommand, timeout time.Duration) error {
+//RunCommandContext run a ssh command but with a context, so can be cancel when is needed
```

Comment:
runs an SSHCommand with a provided context so that the command can be cancelled if needed.